.cspell.json

```diff
@@ -4,16 +4,22 @@
   "cache": {
     "useCache": true
   },
-  "ignorePaths": ["CHANGELOG.md"],
-  "import": ["@cspell/dict-fr-fr/cspell-ext.json"],
+  "ignorePaths": [
+    "CHANGELOG.md"
+  ],
+  "import": [
+    "@cspell/dict-fr-fr/cspell-ext.json"
+  ],
   "words": [
     "adjustednumberoflikes",
     "applitools",
     "ARROWUP",
     "Artemia",
     "Artemiidae",
     "attachtocase",
+    "bazz",
```

Comment:
The linter was blocking me from committing unless I added this to the vocabulary of the project 😅

Reply:
That is because all modified files are run through lint-staged and the spell checker was added recently - so it picks up issues in existing files which are staged.